tests/unit/test_pos.py

```diff
@@ -133,11 +133,21 @@ def test_authors(generated_conference_paper):
     expected_authors = [
         {
             'full_name': 'El-Khadra, Aida',
-            'affiliations': [{'value': 'INFN and Universit\xe0 di Firenze'}],
+            'raw_affiliations': [
+                {
+                    'source': 'pos',
+                    'value': 'INFN and Universit\xe0 di Firenze',
+                },
+            ],
```

Comment:
@michamos, is this right? https://github.com/inspirehep/inspire-next/issues/3193 was talking just about arXiv papers, but I don't see why we should trust an affiliation from PoS to be normalized how we want it to be.